_common.scss

```diff
@@ -311,16 +311,8 @@
         }
     }
 
-    .mat-stepper-horizontal,
-    .mat-stepper-vertical {
-        .mat-form-field:not([blui-input]) {
-            &.mat-form-field-appearance-standard,
-            &.mat-form-field-appearance-legacy {
-                .mat-form-field-infix {
-                    margin-top: -1rem;
-                }
-            }
-        }
+    .mat-horizontal-content-container, .mat-vertical-content {
+        padding: 8px 24px 24px 24px;
```

Comment:
I think we've been keeping vertical padding in px but horizontal padding in rem to account for different screen sizes.  Try this: 

`padding: 8px 1.5rem 24px 1.5rem`